From: Andrew Morton <akpm@linux-foundation.org>
Subject: kernel-reboot-explicitly-notify-if-halt-occurred-instead-of-power-off-fix
Date: Mon Nov  6 08:22:24 AM PST 2023

tweak comment text

Cc: Dongmin Lee <ldmldm05@gmail.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 kernel/reboot.c |    7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

--- a/kernel/reboot.c~kernel-reboot-explicitly-notify-if-halt-occurred-instead-of-power-off-fix
+++ a/kernel/reboot.c
@@ -59,9 +59,10 @@ struct sys_off_handler {
 };
 
 /*
- * This variable is used to indicate if a halt initiated instead when
- * reboot call is invoked with LINUX_REBOOT_CMD_POWER_OFF, but system
- * cannot be powered off. This allowes kernel_halt() to notify that.
+ * This variable is used to indicate if a halt was initiated instead of a
+ * reboot when the reboot call was invoked with LINUX_REBOOT_CMD_POWER_OFF, but
+ * the system cannot be powered off. This allowes kernel_halt() to notify users
+ * of that.
  */
 static bool poweroff_fallback_to_halt;
 
_
